ping.py

```diff
@@ -63,19 +65,121 @@ def get_ident(): return 0
 MAX_SLEEP = 1000
 
 
-class MStats:
-    thisIP = "0.0.0.0"
-    pktsSent = 0
-    pktsRcvd = 0
-    minTime = 999999999
-    maxTime = 0
-    totTime = 0
-    avrgTime = 0
-    fracLoss = 1.0
+# noinspection PyAttributeOutsideInit
```

Comment:
This is to make delinters (e.g., PyCharm) to shut up about properties set outside init. Without this, PyCharm "blew up" with a HUGE number of protests 😑